packages/block-editor/src/components/block-popover/index.js

```diff
@@ -88,6 +93,20 @@ function BlockPopover(
 
 		return {
 			getBoundingClientRect() {
+				if ( isZoomOut() ) {
+					const selectedRect =
+						getVisibleElementBounds( selectedElement );
+					const parentRect = getVisibleElementBounds(
+						selectedElement.parentElement
+					);
+					return new window.DOMRectReadOnly(
+						parentRect.left,
```

Comment:
`parentRect` is the element represented by `sectionRootClientId`? I'm not immediately clear from the code what the "parent" relationship is.